### PR TITLE
chore: update bootc extension to 1.6.0

### PR DESCRIPTION
**Pull request by cdrage**
chore: update bootc extension to 1.6.0

Updates bootc extension to 1.6.0

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

**Comment by cdrage** [comment]
**DO NOT MERGE UNTIL QE APPROVAL**

**Comment by benoitf** [comment]
yes it'll still be ok

File: static/api/extensions.json [hunk 1]
```diff
@@ -468,6 +468,27 @@
       "categories": ["Containers"],
       "keywords": ["bootc", "bootable", "OS", "disk"],
       "versions": [
+        {
+          "version": "1.6.0",
+          "podmanDesktopVersion": "^1.12.0",
```
Comment (benoitf) [inline static/api/extensions.json:473]: should we bump the required version there to a more recent one ? like 1.14 ?

**Review by benoitf — APPROVED** [review]
approving, needs to be approved by QE before merging

**Comment by cdrage** [comment]
@benoitf Approved by QE from Slack, merging!

**Pull request merged**